.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--
+Thank you for your pull request. Please review the below requirements.
+
+Contributor guide: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
+-->
+
+##### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] `npm install test` passes
```

Comment:
I meant `npm install && npm test`, which is the same as `npm install test` (also the same as `npm it`, but you suggested we favour readability, which is fine).

I guess `npm test` will fail if you haven't done an `npm install` first, but better safe than sorry right (you might have messed up the `package.json` since the last time you did an `npm install`).